### PR TITLE
Add json lanuguage server(vscode-json-languageserver)

### PR DESCRIPTION
**Pull request by h-michael**
test on [this repository](https://github.com/fge/sample-json-schemas).

Should fix https://github.com/vscode-langservers/vscode-json-languageserver-bin/issues/3

### CODE REVIEW

File: lua/nvim_lsp/jsonls.lua [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        -- Try to preserve any additional args from upstream changes.
+        new_config.cmd[1] = install_info.binaries[bin_name]
+      else
+        new_config.cmd = {install_info.binaries[bin_name]}
+      end
+    end
+  end;
+  docs = {
+    -- this language server config is in VSCode built-in package.json
+    package_json = "https://raw.githubusercontent.com/microsoft/vscode/master/extensions/json-language-features/package.json";
```
Comment (justinmk) [inline lua/nvim_lsp/jsonls.lua]: Use `vscode` key?

Comment (h-michael) [reply lua/nvim_lsp/jsonls.lua]: I wrote it in the code comment, but this extension is built-in to vscode, so it is not distributed as a package.

**Comment by justinmk** [comment]
needs a rebase (`skeleton` was renamed on master).

Is this ready otherwise? Why still in draft?

**Comment by justinmk** [comment]
if this is working (are you using it?) let's merge it rather than have it sit around. can always improve it later.

**Comment by justinmk** [comment]
I wonder why docgen didn't create a doc commit?

**Comment by h-michael** [comment]
@justinmk see #110

**Pull request merged**